apps/versions/tests.py

```diff
@@ -833,6 +840,22 @@ def test_success(self):
         assert self.addon.current_version
         self.assert_served_by_mirror(self.client.get(self.latest_url))
 
+    def test_beta(self):
+        r = self.client.get(self.latest_beta_url)
```

Comment:
I know you've seen quite a few tests like that (using one letter variables), but please don't do that. We try to write new tests "the good way", not using one letter variables.

Eg: ```response = self.client.get(self.latest_beta_url)```